portlayer/vsphere/storage/parent.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
This file could use a very short comment explaining why we are doing this, what we are doing here and what are expectations and assumptions.